charts/galoy/values.yaml

```diff
@@ -708,23 +715,170 @@ kratos:
     automigration:
       enabled: true
     config:
-      session:
-        lifespan: "720h" # 1 month
-        earliest_possible_extend: "96h" # 4 days
+      secrets:
+        default:
+          - PLEASE-CHANGE-ME-I-AM-VERY-INSECURE
+
+      version: v0.11.0
+
+      dsn: postgresql://kratos-pg:kratos-pg@postgresql/kratos-pg
+
+      serve:
+        public:
+          base_url: http://127.0.0.1:4433/
+          cors:
+            enabled: true
+            allow_credentials: true
+            allowed_origins:
+              - http://127.0.0.1:4433
+              - http://0.0.0.0:4434
+              - http://localhost:3000
+              - http://127.0.0.1:3000
+            allowed_methods:
+              - POST
+              - GET
+              - PUT
+              - PATCH
+              - DELETE
+            allowed_headers:
+              - Authorization
+              - Cookie
+              - Content-Type
+              - X-Session-Token
+            exposed_headers:
+              - Content-Type
+              - Set-Cookie
+            debug: true
+
+        admin:
+          base_url: http://0.0.0.0:4434/
+
+      selfservice:
+        default_browser_return_url: http://localhost:3000/
+        allowed_return_urls:
+          - http://127.0.0.1:4002/
+          - http://localhost:3000/
+      
+        methods:
+          oidc:
+            enabled: false
+          webauthn:
+            enabled: false
+          totp:
+            enabled: true
+          password:
+            enabled: true
+      
+          code:
+            enabled: true
+            config:
+              # Defines how long the verification or the recovery code is valid for (default 1h)
+              lifespan: 15m
+      
+        flows:
+          error:
+            ui_url: http://127.0.0.1:4002/error
+      
+          settings:
+            ui_url: http://127.0.0.1:4002/settings
+            privileged_session_max_age: 15m
+      
+            required_aal: aal1
+      
+            after:
+              profile:
+                hooks:
+                  - hook: web_hook
+                    config:
+                      # url: http://e2e-tests:4002/auth/after_settings_hooks
+                      url: http://invalid-because-we-dont-want-profile-to-be-updated
+                      method: POST
+                      body: file:///home/ory/body.jsonnet
+                      auth:
+                        type: api_key
+                        config:
+                          name: Authorization
+                          value: The-Value-of-My-Key
```

Comment:
hey @openoms , i have been troubleshooting in my local dev charts environment and I think we need to read this value from the `callback_api_key` from here https://github.com/GaloyMoney/charts/blob/main/dev/galoy/main.tf#L314. I am not quite sure how to configure it here in the yaml file.